sdk/cosmos/azure-cosmos/samples/test_axq.py

```diff
@@ -0,0 +1,82 @@
+import time
+import random
+
+import unittest
+from unittest.mock import Mock, MagicMock, patch
+
+from azure.cosmos.cosmos_client import CosmosClient as SyncClient
+from azure.cosmos.partition_key import PartitionKey
+
+connstring = ""
+DATABASE_ID = 'integrateddbbb'
+CONTAINER_ID = 'containerid'
+
+import uuid
+
+
+def get_test_item():
+    async_item = {
+        'id': 'Async_' + str(uuid.uuid4()),
+        'test_object': True,
+        'lastName': 'Smith',
+        'attr1': random.randint(0, 50)
+    }
+    return async_item
+
+
+def quick_test():
+    print("starting")
+    client = SyncClient.from_connection_string(conn_str=connstring)
+    db = client.create_database_if_not_exists(DATABASE_ID)
+    cont = db.create_container_if_not_exists(CONTAINER_ID, partition_key=PartitionKey(path="/id"))
+    body = get_test_item()
+    cont.create_item(body=body)
+
+    @patch.object(cont.client_connection, '_CosmosClientConnection__Get')
+    def get_headers():
+        cont.read_item(item=body["id"], partition_key=body["id"], max_integrated_cache_staleness_in_ms=(1000 * 60))
+
+    get_headers()
```

Comment:
where are we validating the headers value?

Reply:
we're not, I just uploaded this in case Annie wanted to try something since she messaged me, it's not done